### ISSUE TITLE
Automatic Installation doesn't work well on windows OS

### ISSUE THREAD

**Issue by sidiqaldi** [protonemedia/laravel-splade#8]
In the command installation section there is a string search using `PHP_EOL`

```
 $search = 'public function register()' . PHP_EOL . '    {';

 $search = 'protected $routeMiddleware = [' . PHP_EOL;
```

It doesn't run well on windows OS because by default `Http/Kernel.php` and `Exceptions/Handler.php` files use `\n` .

while for `PHP_EOL` on windows use `\r\n`.

the result of the automatic installation on windows:
`Http/Kernel.php`
![image](https://user-images.githubusercontent.com/15923784/180606355-1c9025c2-87df-4b2e-9a41-f4b59bc6346e.png)
`Exceptions/Handler.php` 
![image](https://user-images.githubusercontent.com/15923784/180606364-8715786a-533b-4372-aaa3-b3cea2b61671.png)

**Comment by pascalbaljet** [comment]
Thanks for notifying me! What is your development environment? Are people still using Xampp on Windows? (I'm not a frequent Windows user)

**Comment by sidiqaldi** [comment]
I'm using Laragon. It's basically Xampp with some automation.

**Comment by pascalbaljet** [comment]
Fixed in [v0.2.2](https://github.com/protonemedia/laravel-splade/releases/tag/0.2.2)
